tests/integration/functions/runtime_context/typed_object_parameters_test.ts

```diff
@@ -0,0 +1,420 @@
+import {
```

Comment:
File named typed object parameter or typed object property?

Reply:
Ooof, tough one haha. Either works for me. What do you prefer?

Reply:
Property since it's consistent with the method name

Reply:
Done in 9e1f9fd